lib/svgo/coa.js

```diff
@@ -146,7 +146,7 @@ module.exports = require('coa').Cmd()
             !opts.string &&
             !opts.stdin &&
             !opts.folder &&
-            process.stdin.isTTY !== false
+            Boolean(process.stdin.isTTY)
```

Comment:
This was done for a reason. In Windows CLI `isTTY` is `undefined`, that why it's being comparing to `false` here. Another solution is needed.